tests/fixtures/tumor-variant-freq-study-result.yaml

```diff
@@ -0,0 +1,117 @@
+id: cancer_hotspots:CIC-R215
+type: TumorVariantFrequencyStudyResult
+name: Overall Tumor Variant Frequencr for CIC R215
+sourceDataSet:
+  type: DataSet
+  name: Cancer Hotspots
+  version: "2"
+specifiedBy:
+  type: Method
+  reportedIn:
+    type: Document
+    doi: 10.1158/2159-8290.CD-17-0321
+    name: Accelerating Discovery of Functional Mutant Alleles in Cancer
+affectedSampleCount: 25
+totalSampleCount: 24592
+affectedFrequency: 0.001016590761223162
+focusVariant:
+  type: CategoricalVariant
+  name: CIC R215
```

Comment:
Hrm. Are we not able to represent these types of variants in Cat-VRS yet? @DanielPuthawala do we have an example of a location + missense protein sequence consequence categorical variant?

Reply:
Oh, I was just keeping it simple for this example since there should be other categorical variant examples in Cat-VRS